package.json

```diff
@@ -24,14 +24,11 @@
     "build": "ember build --environment=production",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:eslint-check": "eslint --print-config .eslintrc.js | eslint-config-prettier-check",
-    "lint:fix": "yarn lint:js --fix",
```

Comment:
I'm not sure what would happen if files are changed in the middle of a CI run.

I think it's better to remove this script (approach taken here). As an alternative, I'd name this script differently (e.g. `fix:js`) so that `yarn lint` will only lint files and not change them.